packages/manager-components/src/components/table/table.mock.ts

```diff
@@ -0,0 +1,46 @@
+export const mockColumns = [
+  {
+    header: 'First Name',
+    accessoryKey: 'firstName',
+  },
+  {
+    header: 'Last Name',
+    accessoryKey: 'lastName',
+  },
+  {
+    header: 'Genre',
+    accessoryKey: 'genre',
+  },
+  {
+    header: 'Role',
+    accessoryKey: 'role',
+  },
+  {
+    header: 'Date Of Birth',
+    accessoryKey: 'birth',
+  },
+];
```

Comment:
**suggestion:** To reduce rework while migration to ODS Datagrid, please use the same attributes that are defined by the ODS components, https://ovh.github.io/design-system/latest/?path=/story/ods-components-layout-datagrid-organism-usage-guidelines--page.
```suggestion
export const mockColumns = [
  {
    title: 'First Name',
    field: 'firstName',
  },
  {
    title: 'Last Name',
    field: 'lastName',
  },
  {
    title: 'Genre',
    field: 'genre',
  },
  {
    title: 'Role',
    field: 'role',
  },
  {
    title: 'Date Of Birth',
    field: 'birth',
  },
];
```

Reply:
'Tanstack query' use this object format, 'title' and 'field' was the format of 'tabulator' , we will not use 'tabulator' in the future